src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  color: $main-color;
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+
+  @include transition(all);
```

Comment:
The mixin `transition(all)` should be defined in your mixins file. Ensure that it is correctly defined and imported if not already.